src/cli/utils/onedrive_test.go

```diff
@@ -26,6 +26,7 @@ func (suite *OneDriveUtilsSuite) TestIncludeOneDriveRestoreDataSelectors() {
 		containsOnly      = []string{"contains"}
 		prefixOnly        = []string{"/prefix"}
 		containsAndPrefix = []string{"contains", "/prefix"}
+		onlySlash         = []string{"/"}
```

Comment:
If we compare on a const, we should also test on a const.
```suggestion
		onlySlash         = []string{path.PathSeparator}
```